manuscript/02-bestandteile-eines-modellgetriebenen-designs.md

```diff
@@ -1,177 +1,180 @@
+seiner Definition im Modell. Halte die Klassendefinition einfach und
 konzentriere dich auf die Kontinuität des Lebenszyklus und
-Identität.**
+die Identität.**
 
 **Definiere ein Möglichkeit zur Unterscheidung jedes Objekts
-unabhängig von seiner Form oder Geschichte.  Achte auf Anforderungen,
-die einen Abgleich von Objekten über Attribute erfordern.  Definiere
+unabhängig von seiner Form oder Geschichte. Achte auf Anforderungen,
+die einen Abgleich von Objekten über Attribute erfordern. Definiere
 eine Operation, die garantiert ein eindeutiges Ergebnis für jedes
-Objekt liefert, möglicherweise durch Anhängen eines garantiert
-eindeutiges Symbols.  Dieser Identifikationsmechanismus kann von außen
-kommen, oder es kann ein beliebiger Identifikator sein, der vom und
-für das System erstellt wird, aber es muss der Unterscheidung der
-Identität im Modell passen.**
+Objekt liefert, möglicherweise durch Ergänzen eines garantiert
+eindeutigen Attributes. Dieses Attribut zur Identifikation kann von 
+außen kommen, oder es kann ein beliebiger Identifikator sein, der vom 
+und für das System erstellt wird, muss aber zu den 
+Identitätsunterschieden im Modell passen.**
 
 **Das Modell muss definieren, was es bedeutet, dasselbe Ding zu
 sein.**
 
-## Value Object {#value-object}
+(auch bekannt als Reference Objects, dt.: Referenzobjekte)
 
-*Dt.: Wertobjekt*
+## Value Objects {#value-object}
 
-Einige Objekte beschreiben oder berechnen eine Eigenschaft eines Dins.
+*dt.: Wertobjekte*
+
+Einige Objekte beschreiben oder berechnen eine Eigenschaft eines Dings.
 
 Viele Objekte haben keine konzeptionelle Identität.
 
 Die Identität von [Entities](#entity) zu verfolgen ist unerlässlich,
-aber andere Objekte eine Identität zuzuweisen, kann die Systemleistung
+aber anderen Objekten eine Identität zuzuweisen, kann die Systemleistung
 beeinträchtigen, mehr Analysetätigkeiten erwzingen und das Modell
-durcheinander bringen, indem alle Objekte gleich aussehen.
-Software-Design ist ein ständiger Kampf mit Komplexität.  Wir müssen
-differenzieren, damit eine besondere Behandlung nur dann erfolgt, wenn
-sie notwendig ist.
+durcheinander bringen, weil alle Objekte gleich aussehen.
+Softwareentwurf ist ein ständiger Kampf gegen Komplexität. Wir müssen
+Unterscheidungen machen, damit eine besondere Behandlung nur dann 
+erfolgt, wenn sie notwendig ist.
 
 Wenn wir diese Kategorie von Objekten jedoch nur als die Abwesenheit
 von Identität verstehen, haben wir nicht viel zu unserem
-Werkzeugkasten oder Vokabular hinzugefügt.  Tatsächlich haben diese
+Werkzeugkasten oder Vokabular hinzugefügt. Tatsächlich haben diese
 Objekte eigene Eigenschaften und eine eigene Bedeutung für das Modell.
 Dies sind Objekte, die Dinge beschreiben.
 
 Daher:
 
 **Wenn du dich nur um die Attribute und die Logik eines Elements des
-Modells kümmern, klassifiziere es als [Value Object](#value-object).
+Modells kümmerst, klassifiziere es als Wertobjekt.
```

Comment:
Meinetwegen können wir die Referenz auf das Pattern entfernen, aber IMHO sollten wir den Namen englisch lassen.

Reply:
Ok, das würde für mich dann bedeuten, dass wir bei den Mustern nie die deutsche Übersetzung verwenden im Text, richtig?

Reply:
Fixed in 234944d

Reply:
Ja, so sehe ich es. Damit ist klar, dass es um das Muster geht. Wir könnten überlegen, das Muster zusätzlich *kursiv* oder **fett** zu setzen, wenn es kein Link ist.